src/gwt/www/js/focus-visible.min.js

```diff
@@ -0,0 +1,2 @@
+//# sourceMappingURL=focus-visible.min.js.map
```

Comment:
Should we also commit the .map file and the original .js file for debugging? (if not, should we remove the source mapping?)